repository/src/main/resources/db/migration/V0002__RDM-10041.sql

```diff
@@ -0,0 +1,19 @@
+insert into field_type (created_at, reference, version, base_field_type_id)
+values (now(), 'CaseLocation', 1,
+    (select id from field_type where reference = 'Complex'
+        and jurisdiction_id is null
+        and version = (select max(version)
+        from field_type where reference = 'Complex'
+        and jurisdiction_id is null
+        and base_field_type_id is null))
+);
+
+insert into complex_field (reference, label, security_classification, field_type_id, complex_field_type_id)
+values ('RegionId', 'RegionId', 'PUBLIC',
```

Comment:
can you ask the BA what is the label they would want to appear on the UI for these 2 subfields? 
Not sure whether `RegionId` is a very human readable label. Wouldn't `Region id` be better? same for the other field